Make.config

```diff
@@ -30,18 +28,12 @@ else
 CURRENT_BRANCH:=$(BRANCH_NAME)
 endif
 
-# TODO: reset to 0 after major/minor version bump (SRO) and increment for service releases and previews
-# Note: if not reseted to 0 we can skip a version and start with .1 or .2
-PACKAGE_VERSION_REV=0
-IOS_PACKAGE_VERSION_REV=$(PACKAGE_VERSION_REV)
-
 IOS_PRODUCT=Xamarin.iOS
 IOS_PACKAGE_NAME=Xamarin.iOS
 IOS_PACKAGE_NAME_LOWER=$(shell echo $(IOS_PACKAGE_NAME) | tr "[:upper:]" "[:lower:]")
-# NEVER customize IOS_PACKAGE_VERSION itself, other parts (mtouch, web updater) are using the IOS_PACKAGE_VERSION_* variables
-IOS_PACKAGE_VERSION=12.2.$(IOS_PACKAGE_VERSION_REV).$(IOS_COMMIT_DISTANCE)
 IOS_PACKAGE_VERSION_MAJOR=$(word 1, $(subst ., ,$(IOS_PACKAGE_VERSION)))
 IOS_PACKAGE_VERSION_MINOR=$(word 2, $(subst ., ,$(IOS_PACKAGE_VERSION)))
+IOS_PACKAGE_VERSION_REV:=$(word 3, $(subst ., ,$(IOS_PACKAGE_VERSION)))
```

Comment:
`:=$`?

Reply:
it can be overridden=, e.g. `IOS_PACKAGE_VERSION_REV=9 make`
I don't think it's required here, but it's not harmful

Reply:
`:=` means "calculate value once and store it in the variable"
`=` means "store string as-is, and evaluate it every time the variable is used"

The latter can be slow if the variable is used many times, in particular if it calls an external process.